RELEASE_PROCESS.md

```diff
@@ -297,3 +317,13 @@ spelled with complete minutiae.
       1. A blue box will appear above the job list with a right side drop
          -down. Click the drop-down and specify the version number in the
          dialog that appears
+
+1. Annouce the release
+      1. For major and minor releases, write a blog and publish it to blog.podman.io
+         Highlight key features and important changes or fixes. Link to the github release.
+         Make sure the blog is properly  tagged with the Announcement, Release, and Podman tags,
```

Comment:
```suggestion
         Make sure the blog post is properly tagged with the Announcement, Release, and Podman tags,
```

The "blog" is the blogs.podman.io site in its entirety.  Each article in the blog is a post  (Yeah, I struggle with this one too)..